flatpak/meson/postinstall.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python3
```

Comment:
This file should be part the upstream source code?

Reply:
Thanks. Will remove it.